### DISCUSSION TITLE
Initialization of dynamic simulations

### DISCUSSION THREAD

**Discussion by lyyc199586** [idaholab/moose#25257] [Q&A Modules: General]
In `timeintegrator` (for example `CentralDifferece`), if I don't explicitly specify the initial velocities, does MOOSE treat the initial velocities as zero?

I know you can set `inactive_tsteps=1` to switch off the velocity and acceleration calculation at the first time step, my question is if I don't do that, what value did MOOSE assign to the initial velocities?

**Comment by GiudGiud** [comment]
Variables are initialized to 0 if they are not otherwise initialized.

**Reply by lyyc199586** [reply]
great, got it
